subprojects/docs/src/snippets/dependencyManagement/dependencyVerification-disablingVerification/kotlin/build.gradle.kts

```diff
@@ -29,7 +29,7 @@ configurations {
 tasks.register("checkDependencies") {
     inputs.files(myPluginClasspath)
     doLast {
-        println(myPluginClasspath.files)
+        println(inputs.files)
```

Comment:
Note that the change to `input.files` is not equivalent to the original `myPluginClasspath.files`, as the configuration's `files` is a `Set<File>`, while `input.files` is a `FileCollection` whose contents won't be printed by `println` (it will look like `task ':checkDependencies' input files`).

Could you please fix this so that the behavior is kept as it was before?

Reply:
Done!